servicetalk-concurrent-api/docs/modules/ROOT/pages/blocking-implementation.adoc

```diff
@@ -38,22 +40,54 @@ Part 2. defines all the interactions using the ReactiveStreams specifications, i
 
 ServiceTalk concurrency APIs defines which thread will be used by any asynchronous source for Part 2.
 
-== Executor and asynchronous sources
-
-ServiceTalk enforces that each asynchronous source **MUST** be associated with an `Executor` which is used to interact
-with its `Subscriber`. The below diagram illustrates the interaction between an asynchronous source, its `Subscriber`,
+== Task Offloading
+
+ServiceTalk uses Netty for network I/O and the Netty implementation uses a fixed number of I/O threads for executing
+network operations. To maximize throughput and minimize latency it is important to ensure that at least one I/O thread
+is always ready to respond immediately to network activity. One approach to ensure I/O thread availability is to
+carefully limit the scope of work done by I/O threads and, when practical, perform any other necessary work on some
+other non-precious thread. Moving work from I/O threads to other threads is called “offloading” and is a core technique
+used by ServiceTalk.
+
+ServiceTalk will, by default, execute most application code on an offloaded “safe” thread. If the application developer
```

Comment:
most -> all? 
"most" rises a question: what is not offloaded?

Reply:
I will clarify.